packages/safe-ds-lang/tests/helpers/testResources.ts

```diff
@@ -59,13 +59,13 @@ export const listTestSafeDsFiles = (rootTestResourceName: TestResourceName): URI
 };
 
 /**
- * Lists all Python files in the given root directory.
+ * Lists all Python files and JSON files in the given root directory.
  *
  * @param rootTestResourceName The resource name of the root directory.
  * @return URIs of the discovered Python files.
  */
-export const listTestPythonFiles = (rootTestResourceName: TestResourceName): URI[] => {
-    const pattern = `**/*.py`;
+export const listTestPythonJsonFiles = (rootTestResourceName: TestResourceName): URI[] => {
+    const pattern = `**/*.{json,py}`;
```

Comment:
```suggestion
export const listTestFilesWithExtensions = (rootTestResourceName: TestResourceName, extensions: string[]): URI[] => {
    const pattern = `**/*.{${extensions.join}}`;
```

I think this more general function is probably easier to understand. The docstring also needs to be adjusted. You can leave the function for Safe-DS files, but maybe call `listTestFilesWithExtensions` internally.